Mathlib/Algebra/Free.lean

```diff
@@ -85,11 +86,11 @@ theorem hom_ext {β : Type v} [Mul β] {f g : FreeMagma α →ₙ* β} (h : f 
 
 end FreeMagma
 
+#adaptation_note /-- around nightly-2024-02-25, we need to write `mul x y` in the second pattern,
+instead of `x * y`. --/
 /-- Lifts a function `α → β` to a magma homomorphism `FreeMagma α → β` given a magma `β`. -/
 def FreeMagma.liftAux {α : Type u} {β : Type v} [Mul β] (f : α → β) : FreeMagma α → β
   | FreeMagma.of x => f x
-  -- Adaptation note: around nightly-2024-02-25, we need to write `mul x y` in the pattern here,
```

Comment:
Writing #adapation_note here errors, hence I moved this comment before the function.